koku/masu/api/views.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2018 Red Hat, Inc.
+# Copyright 2019 Red Hat, Inc.
```

Comment:
Please don't modify the Copyright stanza. It is not something that gets updated after it has been written.

It needs to reflect the *first* year the file was written, not the current year.

Reply:
This is a new file, so I think 2019 is ok here. Right?  Looks like GitHub saw this as a file name change from worker.py to views.py?

Reply:
If it's a new file, why is there a diff of changed lines?

Reply:
Yeah I really can't explain why it's showing up like this, but bottom line is that this file is new.  All related to getting views setup in Django